app/views/handlers/individual_response.py

```diff
@@ -0,0 +1,307 @@
+
+    def _render_block(self):
+        transformed_block = transform_variants(
+            self._block_definition,
+            self._schema,
+            self._questionnaire_store.metadata,
+            self._questionnaire_store.answer_store,
+            self._questionnaire_store.list_store,
+            None,
+        )
+
+        return self.placeholder_renderer.render(transformed_block, self._list_item_id)
+
+
+class IndividualResponseHowHandler(IndividualResponseHandler):
+    block_definition: Mapping = {
+        "type": "IndividualResponse",
+        "id": "individual-response",
+        "question": {
+            "type": "Question",
+            "id": "individual-response-how",
+            "title": {
+                "text": lazy_gettext(
+                    "How would you like <em>{person_name}</em> to receive a separate census?"
+                ),
+                "placeholders": IndividualResponseHandler._person_name_placeholder,
+            },
+            "description": lazy_gettext(
+                "<p>For someone to complete a separate census, we need to send them an individual access code.</p><p>Select how to send access code</p>"
+            ),
+            "answers": [
+                {
+                    "type": "Radio",
+                    "id": "individual-response-how-answer",
+                    "mandatory": True,
+                    "options": [
+                        {
+                            "label": lazy_gettext("Post"),
+                            "value": "Post",
+                            "description": lazy_gettext(
+                                "We can only send this to an unnamed resident at the registered household address"
+                            ),
+                        }
+                    ],
+                }
+            ],
+        },
+    }
+
+    def __init__(self, schema, questionnaire_store, language, form_data, list_item_id):
+        super().__init__(
+            self.block_definition,
+            schema,
+            questionnaire_store,
+            language,
+            form_data,
+            list_item_id,
+        )
+
+    def handle_get(self):
+        previous_location_url = url_for(
+            "individual_response.request_individual_response",
+            list_item_id=self._list_item_id,
+        )
+
+        return render_template(
+            "individual_response/how",
+            language=self._language,
+            content=self.get_context(),
+            previous_location_url=previous_location_url,
+        )
+
+
+class IndividualResponsePostAddressConfirmHandler(IndividualResponseHandler):
+    block_definition: Mapping = {
+        "type": "IndividualResponse",
+        "question": {
+            "type": "Question",
+            "id": "individual-response-post-confirm",
+            "title": {
+                "text": lazy_gettext(
+                    "Do you want to send an individual access code to <em>{person_name}</em> by post?"
```

Comment:
In the prototype, this is `access code for` not `access code to`. And the person name is not surrounded by `em` tags, so worth checking if it should be.